src/work-allocation/components/case-manager-filter/case-manager-filter.component.ts

```diff
@@ -154,7 +154,7 @@ export class CaseManagerFilterComponent implements OnInit, OnDestroy {
   public ngOnInit(): void {
     this.appStoreSub = this.appStore.pipe(select(fromAppStore.getUserDetails)).subscribe(
       (userDetails) => {
-        const isLegalOpsOrJudicialRole = userDetails.userInfo && userDetails.userInfo.roles ? AppUtils.getUserRole(userDetails.userInfo.roles) : null;
+        const isLegalOpsOrJudicialRole = AppUtils.getUserRole(userDetails?.userInfo?.roles || null);;
```

Comment:
AppUtils.getUserRoles() doesn't handle a null input properly, so this will throw an exception. Should be:
 AppUtils.getUserRole(userDetails?.userInfo?.roles || [] );

i.e. passing an empty array if the userDetails/roles are undefined